app/routes.py

```diff
@@ -1 +1,245 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, request, abort
+from datetime import datetime
+import requests
+import json
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+task_list_bp = Blueprint("task_list", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals_list", __name__, url_prefix="/goals")
+#validating the task_id
+def validate_task(task_id):
+    try:
+        task_id = int(task_id)
+    except:
+        abort(make_response({"message": f"Task {task_id} invalid"}, 400))
+    task = Task.query.get(task_id)
+    if not task:
+        abort(make_response({"details": "Invalid Data"}, 404))
+    return task
+
+#validate goal_id
+def validate_goal(goal_id):
+    try:
+        goal_id = int(goal_id)
+    except:
+        abort(make_response({"message": f"Goal {goal_id} invalid"}, 400))
+    goal = Goal.query.get(goal_id)
+    if not goal:
+        abort(make_response({"details": "Invalid Data"}, 404))
+    return goal
+    
+# create tasks
+@task_list_bp.route("", methods=["POST"])
+def post_task():
+    request_body = request.get_json()
+    if "title" not in request_body or "description" not in request_body:
+        return make_response({"details": "Invalid data"}, 400)
+    new_task = Task(
+        title = request_body["title"],
+        description = request_body["description"],
+        # completed_at = request_body["completed_at"]
+    )
+    db.session.add(new_task)
+    db.session.commit() 
```

Comment:
Great job using `db.session.<method-name>`. Session gives us access to the follow:

- `db` is our app’s instance of SQLAlchemy.
- `session` represents our active database connection.
- By referencing `db.session` we can use SQLAlchemy’s methods to perform tasks like:
    - committing a change to a model
    - storing a new record of a model
    - deleting a record.
- By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the task model